pkg/registry/storage/csidriver/strategy_test.go

```diff
@@ -113,21 +113,12 @@ func TestCSIDriverPrepareForCreate(t *testing.T) {
 			name:         "capacity disabled",
 			withCapacity: false,
 		},
-		{
```

Comment:
`withServiceAccountToken` definition can be removed.

Reply:
done